main.js

```diff
@@ -13,7 +15,11 @@ function createWindow () {
   mainWindow = new BrowserWindow({width: 800, height: 600})
 
   // and load the index.html of the app.
-  mainWindow.loadURL(`file://${__dirname}/index.html`)
+  mainWindow.loadURL(url.format({
+    pathname: __dirname + '/index.html',
+    protocol: 'file:',
+    slashes: true
```

Comment:
This option can be removed once Electron runs Node 7+, https://github.com/nodejs/node/commit/336b027411b2f413e02033720bbc4a5f3f0d96bd